src/main/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/IResourceRepository.kt

```diff
@@ -9,6 +9,7 @@ import org.wycliffeassociates.otter.common.data.workbook.ResourceInfo
 interface IResourceRepository : IRepository<Content> {
     fun getResources(collection: Collection, resourceInfo: ResourceInfo): Observable<Content>
     fun getResources(content: Content, resourceInfo: ResourceInfo): Observable<Content>
+    fun getResourcesForCollectionAndChildren(collection: Collection, resourceInfo: ResourceInfo): Observable<Content>
```

Comment:
This function works like the old `getResources` function for collections. It returns resources about the collection (chapter) and its contents (chunks). The functionalities of these methods and their names are subject to debate.

Reply:
I don't think we actually need `getResourcesForCollectionAndChildren`.  I think your new `getResources` behavior is all that's needed now.

Reply:
The only reason I can think that we might want it is if we want to load all of the resources at once for a chapter to calculate progress. But we haven't discussed how we'll do that yet, and this function wouldn't be totally necessary anyway. So I'll remove it.